app/models/map.rb

```diff
@@ -31,27 +31,14 @@ class Map < ActiveRecord::Base
   has_many :annotations, :dependent => :destroy
   belongs_to :user
 
-  has_many :warpables do
-    def public_filenames
-      filenames = {}
-      self.each do |warpable|
-        filenames[warpable.id] = {}
-        sizes = Array.new(Warpable::THUMBNAILS.keys).push(nil)
-        sizes.each do |size|
-          key = size != nil ? size : "original"
-          filenames[warpable.id][key] = warpable.public_filename(size)
-        end
-      end
-      filenames
-    end
-  end
+  has_many :warpables 
 
   def validate
     self.name != 'untitled'
     self.lat >= -90 && self.lat <= 90 && self.lon >= -180 && self.lat <= 180
   end
 
-  # Hash the password before saving the record
+  #Hash the password before saving the record
   def before_create
     self.password = Password::update(self.password) if self.password != ""
   end
```

Comment:
This is supposed to Hash the password but `before_create` is never called????
Map passwords are stored in plain text???
Is this okay???

Reply:
Password needs to be encrypted with one way hash functions. This should be problem as per theory

Reply:
@jywarren Map passwords are not being hashed. They're stored in plain text. 
Security concern?

Reply:
This is quite an old feature, but the problem is that we may have existing maps that use it. Can we confirm that there is no way to /make/ a new passworded map?